source/common/http/conn_manager_impl.cc

```diff
@@ -831,6 +832,10 @@ void ConnectionManagerImpl::ActiveStream::decodeData(ActiveStreamDecoderFilter*
   }
 
   for (; entry != decoder_filters_.end(); entry++) {
+    // if end_stream_ is called for this filter, not to call its decodeData() again
```

Comment:
Can you make this comment more robust to describe the original issue that we are protecting against for future readers? Also, nit, please start sentence with a capital and end with a period.

Reply:
I updated the comment to refer to the issue.